components/match2/wikis/dota2/match_page_template.lua

```diff
@@ -80,40 +80,48 @@ return {
 					</div>
 				</div>
 			</div>
-			<h3>Head-to-Head</h3>
-			<div class="match-bm-lol-h2h">
-				<div class="match-bm-lol-h2h-header">
-					<div class="match-bm-lol-h2h-header-team">{{&opponents.1.iconDisplay}}</div>
-					<div class="match-bm-lol-h2h-stat-title"></div>
-					<div class="match-bm-lol-h2h-header-team">{{&opponents.2.iconDisplay}}</div>
+			<h3>Team Stats</h3>
+			<div class="match-bm-team-stats">
+				<div class="match-bm-team-stats-header">
+					<h4 class="match-bm-team-stats-header-title">{{winnerName}} Victory</h4>
```

Comment:
Should probably be conditioned that there is a winner, eg. in an upcoming or ongoing match it will now say "Victory" before someone has won

Reply:
I changed it to 
`
<div class="match-bm-team-stats-header">
					{{#winnerName}}<h4 class="match-bm-team-stats-header-title">{{winnerName}} Victory</h4>{{/winnerName}}
					{{^winnerName}}<h4 class="match-bm-team-stats-header-title">No winner determined yet</h4>{{/winnerName}}
				</div>
`

What do you think?